sirepo/package_data/static/js/activait.js

```diff
@@ -1233,8 +1233,9 @@ SIREPO.app.directive('imagePreviewPanel', function(requestSender) {
                     }
                 );
             };
-
-            loadImageFile();
+            setTimeout(function(){
```

Comment:
You noted this in your commit and this is a major code smell. What if things are slow and it takes longer than 3s? Instead of doing this we need to structure the code so whatever needs to be ready is ready when this is called. Or make this function somehow protect itself. Lmk if you want to meet and figure out how to do that.